src/DynamoCore/Models/DynamoModel.cs

```diff
@@ -644,9 +647,38 @@ protected DynamoModel(IStartConfiguration config)
             }
 
             // If user skipped analytics from assembly config, do not try to launch the analytics client
+            // or the feature flags client.
             if (!areAnalyticsDisabledFromConfig && !Dynamo.Logging.Analytics.DisableAnalytics)
             {
                 AnalyticsService.Start(this, IsHeadless, IsTestMode);
+
+
+                //run process startup/reading on another thread so we don't block dynamo startup.
+                //if we end up needing to control aspects of dynamo model or view startup that we can't make
+                //event based/async then just run this on main thread - ie get rid of the Task.Run()
```

Comment:
By `if we end up needing to control aspects of dynamo model or view startup`, do you mean a case where we'll need to check for feature flags at some point before the model is instantiated?

Reply:
yes, and where we cannot recreate or reset that aspect of the model or view.